lib/mayaUsd/fileio/translators/translatorMayaReference.h

```diff
@@ -62,6 +62,11 @@ struct UsdMayaTranslatorMayaReference
     static MStatus update(const UsdPrim& prim, MObject parent);
 
 private:
+    static MString namespaceFromPrim(const UsdPrim& prim);
```

Comment:
Nit-pick: if these methods are static and private, you may as well move them to the unnamed namespace in the .cpp file, to reduce recompilation and dependencies.  In a small project like ours, admittedly it doesn't make much difference, but it's still good practice.

Reply:
Those functions are using private static class members.